content/300-guides/200-deployment/201-serverless/300-deploy-to-vercel.mdx

```diff
@@ -32,6 +32,15 @@ Vercel will automatically cache dependencies on deployment. For most application
 
 This will re-generate Prisma Client at build time so that your deployment always has an up-to-date client.
 
+Another option to avoid an outdated Prisma Client is to use [a custom output path](/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path) and check your client into version control.
```

Comment:
```suggestion
Another option to avoid an outdated Prisma Client is to use [a custom output path](/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path) and check your client into version control. This way each deployment is guaranteed to include the correct Prisma Client.
```